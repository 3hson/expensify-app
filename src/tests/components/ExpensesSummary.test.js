import React from 'react';
import {shallow} from "enzyme";
import ExpensesSumary, { ExpensesSummary } from "../../components/ExpensesSummary";

test('should rnder Expenses summary witht 1 expense', ()=>{
    const wrapper = shallow (<ExpensesSummary expenseCount={1} expensesTotal={235} />);
    expect(wrapper).toMatchSnapshot();
})



test('should rnder Expenses summary witht multiple expense', ()=>{
    const wrapper = shallow (<ExpensesSummary expenseCount={23} expensesTotal={222123335} />);
    expect(wrapper).toMatchSnapshot();
})