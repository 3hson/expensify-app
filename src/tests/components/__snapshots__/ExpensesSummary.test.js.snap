// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should rnder Expenses summary witht 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`should rnder Expenses summary witht multiple expense 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $2,221,233.35
  </h1>
</div>
`;
